src/Rhino.Licensing/LicenseType.cs

```diff
@@ -15,20 +15,10 @@ public enum LicenseType
         /// </summary>
         Trial,
 
-        /// <summary>
-        /// Standard license
-        /// </summary>
-        Standard,
-
-        /// <summary>
-        /// For personal use
-        /// </summary>
-        Personal,
-
         /// <summary>
         /// Professional license (subscription)
         /// </summary>
-        Professional,
+        Professional = 4,
```

Comment:
To keep the enum values for the licenses we do use in tact since Standard and Personal were removed, didn't want Professional to suddenly be 2. That being said, I don't think we actually use the values anywhere, I believe it's all the string representations. I could try a build without that set and run through Test Kitchen and what not 🤔

Reply:
Thanks for the info @vexx32. @gep13 and I discovered that it does have the appropriate values according to the intellisense in VSCode .Net Interactive Notebooks. I've also seen it has the right value when cast to an int.

Perhaps adding a test to ensure they cast as expected is beneficial so we can be confident they don't change 🤔

Reply:
Aye, I agree a test is a good idea so that we can be sure we don't break things for our internal projects.

Reply:
I've updated this to include a comment that tests were removed. I've also added a test for the mapping to the proper integer representation. For those tests I purposely added the 4 removed licenses and set their names to include that they're removed so we don't accidentally try using them in the future.